packages/esm-patient-flags-app/src/patient-flags/patient-flags-tag.component.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+import { Tag } from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+import { useFlagsFromPatient, useCurrentPath } from '../hooks/usePatientFlags';
+import styles from './patient-flags.scss';
+import { Route, useLocation, useInRouterContext, useParams } from 'react-router-dom';
+
+interface PatientFlagsTagProps {
+  patientUuid: string;
+}
+
+const PatientFlagsTag: React.FC<PatientFlagsTagProps> = ({ patientUuid }) => {
+  const { t } = useTranslation();
+  const path = useCurrentPath();
+  const [shouldShowTag, setShouldShowTag] = useState(false);
+  const { flags, isflagsLoading, flagsError } = useFlagsFromPatient(patientUuid);
+
+  useEffect(() => {
+    if (path.includes('Summary')) {
+      setShouldShowTag(false);
+    } else {
+      setShouldShowTag(true);
+    }
+  }, [path]);
```

Comment:
It's a bad idea to hard-code this against the path... I'm actually not sure why this is here?

Reply:
adding to https://github.com/openmrs/openmrs-esm-patient-chart/pull/926#discussion_r1071794830 since window.location.pathname isn't guaranteed to change each time the location changes (more so without actual loading) showing the number of flags manifested delays with the user navigating to other parts of the patient dashboard from the patient summary.